test/test.js

```diff
@@ -1203,8 +1203,42 @@ describe('biospecimen', async () => {
                        `Kit status ${status} eligible for replacement kit`);
                 });
             });
-            it('Should prevent participant with a pending home MW kit from obtaining a replacement', () => {
+            it('Should request an initial kit for a participant with no collection details', () => {
                 const data = {
+                    [fieldToConceptIdMapping.firstName]: 'First',
+                    [fieldToConceptIdMapping.lastName]: 'Last',
+                    [fieldToConceptIdMapping.address1]: '123 Fake Street',
+                    [fieldToConceptIdMapping.city]: 'City',
+                    [fieldToConceptIdMapping.state]: 'PA',
+                    [fieldToConceptIdMapping.zip]: '19104',
+                    'Connect_ID': 123456789
+                };
+    
+                const updates = shared.getHomeMWKitData(data);
+                const clonedUpdates = Object.assign({}, updates);
+                delete clonedUpdates[`${fieldToConceptIdMapping.collectionDetails}.${fieldToConceptIdMapping.baseline}.${fieldToConceptIdMapping.bioKitMouthwash}.${fieldToConceptIdMapping.dateKitRequested}`];
+                assert.closeTo
+                    (+new Date(updates[`${fieldToConceptIdMapping.collectionDetails}.${fieldToConceptIdMapping.baseline}.${fieldToConceptIdMapping.bioKitMouthwash}.${fieldToConceptIdMapping.dateKitRequested}`]),
+                    +new Date(), 
+                    60000, 
+                    'Date kit requested is within a minute of test completion'
+                );
+    
+                // console.log('updates', updates);
+                assert.deepEqual(clonedUpdates, {
+                    [`${fieldToConceptIdMapping.collectionDetails}.${fieldToConceptIdMapping.baseline}.${fieldToConceptIdMapping.bioKitMouthwash}.${fieldToConceptIdMapping.kitType}`]: 976461859,
+                    [`${fieldToConceptIdMapping.collectionDetails}.${fieldToConceptIdMapping.baseline}.${fieldToConceptIdMapping.bioKitMouthwash}.${fieldToConceptIdMapping.kitStatus}`]: 728267588,
```

Comment:
I think `976461859` and `728267588` can be replaced with 'fieldToConceptIdMapping.mouthwashKit' and 'fieldToConceptIdMapping.mouthwashKit'